pkg/manager/manager.go

```diff
@@ -176,13 +176,13 @@ type Options struct {
 	// act on the same resources concurrently.
 	LeaderElectionResourceLock string
 
-	// LeaderElectionNamespace determines the namespace in which the leader
-	// election resource will be created.
-	LeaderElectionNamespace string
+	// LeaderElectionLockNamespace determines the namespace of the leader election lock
```

Comment:
While I do agree that the new names are better, renaming these will result in a lot of churn for consumers, I am not sure if that is worth it

Reply:
Yeah totally valid. It doesn't offer any net new improvement.

Reply:
Would leaving a comment in the docs for this be better?  I don't see this getting merged being that the churn associated with this and the complexity @joelanford  mentioned with trying to promote these fields and deprecating others would help either.

Reply:
To echo other's comments, given that the field are self explanatory, maybe not 100% accurate I guess (for historical reasons these have changed the internal meaning); could we add more information on the comment, rather than renaming them?